samples/tab-deeplink/csharp/DeepLinkBot/Views/Home/ExtendedDeepLinks.cshtml

```diff
@@ -0,0 +1,311 @@
+  {
+      margin-left: 1rem;
+      margin-right: 1rem;
+      margin-top: 1rem;
+      padding-top: 0.5rem;
+      font-size: 16px !important;
+  }
+
+  button {
+      margin: 1rem 8.5rem 1rem 11.5rem;
+  }
+
+  .form-container {
+      margin-left: 1rem;
+  }
+
+  .card-body {
+      margin-left: 0.5rem;
+  }
+</style>
+
+<script type="text/javascript">
+  microsoftTeams.app.initialize();
+
+  function selectPeople() {
+      microsoftTeams.people.selectPeople({ setSelected: [], openOrgWideSearchInChatOrChannel: true, singleSelect: false }).then((people) => {
+          if (people) {
+              document.getElementById("Attendee").value = (people.map((p) => p.email)).join(",");
+          }
+      }).catch((error) => {
+          console.log(error);
+      });
+  }
+
+  function onCallDeepLinkButtonClick(callModalities) {
+      microsoftTeams.people.selectPeople({ setSelected: [], openOrgWideSearchInChatOrChannel: true, singleSelect: false }).then((people) => {
+          if (people) {
+              if(microsoftTeams.call.isSupported()) {
+                  microsoftTeams.call.startCall({ targets: people.map((p) => p.email),
+                      requestedModalities: [callModalities]})
+                  .then((result) => {
+                      console.log(result);
+                  }).catch((error) => {
+                          console.log(error);
+                  });
+              }
+              else {
+                  alert("Call isn't supported");
+              }
+          }
+      }).catch((error) => {
+          console.log(error);
+      });
+  }
+
+  function onSchedulingDialogClick() {
+      var subjectInput = document.getElementById("Subject");
+      var startTimeInput = document.getElementById("StartTime");
+      var endTimeInput = document.getElementById("EndTime");
+      var contentInput = document.getElementById("Content");
+      var attendeeInput = document.getElementById("Attendee");
+
+      if (subjectInput.value.trim() !== "" && startTimeInput.value !== "" && endTimeInput.value !== "" && attendeeInput.value !== "") {
+          var meetingDetails =
+          {
+              attendees: attendeeInput.value,
+              content: contentInput.value.trim(),
+              endTime: endTimeInput.value,
+              startTime: startTimeInput.value,
+              subject: subjectInput.value.trim(),
+          };
+
+          //microsoftTeams.calendar.composeMeeting(meetingDetails).
+          //    then((result) => { console.log(result); }).
+          //    catch((error) => { console.log(error); });
+
+          microsoftTeams.app.openLink(`https://teams.microsoft.com/l/meeting/new?subject=${meetingDetails.subject}&startTime=${meetingDetails.startTime}&endTime=${meetingDetails.endTime}&content=${meetingDetails.content}&attendees=${meetingDetails.attendees}`)
+      }
+  }
+
+  function onAppInstallDialogClick() {
+      var appInstallDetails = { appID: "c4c35992-6454-4b49-b8d3-b16ce2e36db2" };
```

Comment:
Can't we read APp ID from appsettings or pass it from Controller method?

Reply:
This is suggested by Wajeed to put static polly app Id.